src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.js

```diff
@@ -41,7 +42,17 @@ const getBaseUrl = (queryString, isAmp) => {
   }
 };
 
-export default ({ type, mediaId, isAmp = false, queryString }) => {
+const handleAvEmbed = ({ mediaPathname }) => {
+  const parsedRoute = parseAvRoute(mediaPathname);
```

Comment:
I wasn't sure if it's best to use the `parseAvRoute` function here. I'm using it to clean up extra slashes and remove the renderer env.

Reply:
We'd probably need to do similar parsing and reconstruction anyway to format the `mediaPath`, because it could contain things like `/vpid/XXXXXXX`. The `parseAvRoute` should give all that data back and can be extended if needed for the upcoming `ws/av-embeds` routing as a well, so I think its ok to use here.